src/main/java/no/unit/nva/cristin/projects/ProjectsWrapper.java

```diff
@@ -0,0 +1,17 @@
+package no.unit.nva.cristin.projects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class ProjectsWrapper {
```

Comment:
a wrapper that will be filled with NvaProject list in later PR instead of ProjectPresentation